doc/api/addons.markdown

```diff
@@ -421,9 +432,9 @@ To test it in JavaScript:
 // test.js
 const addon = require('./build/Release/addon');
 
-var obj1 = addon('hello');
-var obj2 = addon('world');
-console.log(obj1.msg+' '+obj2.msg); // 'hello world'
+const obj1 = addon('hello');
+const obj2 = addon('world');
+console.log(obj1.msg + ', ' + obj2.msg); // 'hello world'
```

Comment:
as long as we're going `const` lets go `console.log(`${obj1.msg}, ${obj2.msg}`);`